pkg/render/layer.go

```diff
@@ -79,30 +76,24 @@ func getRenderValues(inputs ValuesInputs) (interface{}, error) {
 
 	// TODO: get rid of this and just use the ValuesInputs type
```

Comment:
Can we get rid of this now?

Reply:
In my view, ValuesInputs should be a simple way to tell the renderer everything it needs to produce a manifest. Having to structure it in such a way that it can also be used to render custom values into layers goes against this purpose, as accessing values nested in a flavor or params map isn't practical or possible in a template.

However, I do think that the manifestRenderValues struct should be exported and documented so that spec authors know how to write templates, and we can test directly against this struct. I'll make that change and drop the todo.